### PR TITLE
PD-308: Link non-Kubernetes topics to their relevant k8s counterparts

### PR DESCRIPTION
**Pull request by andreipope**
Link non-Kubernetes topics to their relevant k8s counterparts

### CODE REVIEW

**Comment by andreipope** [comment]
@nathan-px Hey Nathan, please review this PR when you get a chance. Thanks!

File: content/install-with-other/datamigration/_index.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ aliases:
   - /cloud-references/migration/migration-pxctl
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of migrating volumes between two Portworx clusters. Please refer to the [Kubemotion with Stork](/portworx-install-with-kubernetes/migration/kubemotion) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/datamigration/_index.md]:
```suggestion
This document presents the **non-Kubernetes** way of migrating volumes between two Portworx clusters. Please refer to the [Kubemotion with Stork](/portworx-install-with-kubernetes/migration/kubemotion) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/application-installs/_index.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ keywords: portworx, PX-Developer, container, Mesos, Mesosphere, Marathon, storag
 weight: 4
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of installing stateful applications with Portworx. Please refer to the [Stateful applications on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/) page, if you want to install stateful applications with Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/application-installs/_index.md]:
```suggestion
This document presents the **DC/OS** method of installing stateful applications with Portworx. Please refer to the [Stateful applications on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/) page if you want to install stateful applications with Portworx on Kubernetes.
```

File: content/install-with-other/dcos/application-installs/cassandra.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ keywords: portworx, px-developer, cassandra, database, cluster, storage
 linkTitle: Cassandra
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of installing Cassandra using Portworx. Please refer to the [Cassandra with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/cassandra/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/application-installs/cassandra.md]:
```suggestion
This document presents the **DC/OS** method of installing Cassandra using Portworx. Please refer to the [Cassandra with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/cassandra/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/application-installs/elastic-search-and-kibana.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ description: Find out how to install the ElasticSearch service on your DCOS clus
 keywords: portworx, container, Mesos, Mesosphere, DCOS, Elasticsearch
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of installing Elastic Search and Kibana with Portworx. Please refer to the [Elastic Search and Kibana with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/elastic-search-and-kibana/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/application-installs/elastic-search-and-kibana.md]:
```suggestion
This document presents the **DC/OS** method of installing Elastic Search and Kibana with Portworx. Please refer to the [Elastic Search and Kibana with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/elastic-search-and-kibana/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/application-installs/kafka.md [hunk 5]
```diff
@@ -4,6 +4,10 @@ description: Portworx can enable persistent data streams for kafka clusters. To
 keywords: portworx, px-developer, cassandra, database, cluster, storage
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of installing Kafka with Portworx. Please refer to the [Kafka and Zookeeper with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/kafka-with-zookeeper) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/application-installs/kafka.md]:
```suggestion
This document presents the **DC/OS** method of installing Kafka with Portworx. Please refer to the [Kafka and Zookeeper with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/kafka-with-zookeeper) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/application-installs/zookeeper.md [hunk 6]
```diff
@@ -4,6 +4,10 @@ description: For help installing and running Zookeeper on DCOS, use the guide fr
 keywords: portworx, container, Mesos, Mesosphere, DCOS, Zookeeper
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of installing Zookeeper with Portworx. Please refer to the [Kafka and Zookeeper with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/kafka-with-zookeeper) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/application-installs/zookeeper.md]:
```suggestion
This document presents the **DC/OS** method of installing Zookeeper with Portworx. Please refer to the [Kafka and Zookeeper with Portworx on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/kafka-with-zookeeper) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/install-azure/_index.md [hunk 7]
```diff
@@ -7,6 +7,10 @@ weight: 3
 noicon: true
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of installing Portworx on the Azure Container Service Engine (ACS). Please refer to the [Azure Container Service (ACS)](/portworx-install-with-kubernetes/cloud/azure/acs/) page, if you want to install Portworx with Kubernetes on the Azure Container Service Engine (ACS).
```
Comment (nathan-px) [inline content/install-with-other/dcos/install-azure/_index.md]:
```suggestion
This document presents the **DC/OS** method of installing Portworx on the Azure Container Service Engine (ACS). Please refer to the [Azure Container Service (ACS)](/portworx-install-with-kubernetes/cloud/azure/acs/) page if you want to install Portworx with Kubernetes on the Azure Container Service Engine (ACS).
```

File: content/install-with-other/dcos/install/_index.md [hunk 8]
```diff
@@ -7,6 +7,10 @@ linkTitle: Install on DCOS
 noicon: true
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of installing Portworx with Mesosphere DC/OS. Please refer to the [Run Portworx with Kubernetes on Mesosphere DC/OS](/install-with-other/dcos/install-dcos-k8s/) page, if you want to install Portworx with Kubernetes on Mesosphere DC/OS.
```
Comment (nathan-px) [inline content/install-with-other/dcos/install/_index.md]:
```suggestion
This document presents the **non-Kubernetes** method of installing Portworx with Mesosphere DC/OS. Please refer to the [Run Portworx with Kubernetes on Mesosphere DC/OS](/install-with-other/dcos/install-dcos-k8s/) page if you want to install Portworx with Kubernetes on Mesosphere DC/OS.
```

File: content/install-with-other/dcos/operate-and-maintain/_index.md [hunk 9]
```diff
@@ -5,3 +5,7 @@ description: Operating and Maintaining helpful tips and how-to!
 keywords: Portworx, storage, containers
 weight: 5
 ---
+
+{{<info>}}
+This document presents the **DC/OS** way of operating and maintaining your Portworx cluster. Please refer to the [Operate and Maintain](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/operate-and-maintain/_index.md]:
```suggestion
This document presents the **DC/OS** method of operating and maintaining your Portworx cluster. Please refer to the [Operate and Maintain](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/operate-and-maintain/authentication.md [hunk 10]
```diff
@@ -6,6 +6,9 @@ weight: 3
 linkTitle: Authorization with PX Clusters
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of enabling authorization for your Portworx cluster. Please refer to the [Authorization](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/operate-and-maintain/authentication.md]:
```suggestion
This document presents the **DC/OS** method of enabling authorization for your Portworx cluster. Please refer to the [Authorization](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/operate-and-maintain/framework-cleanup.md [hunk 11]
```diff
@@ -41,6 +42,9 @@ dcos node ssh --master-proxy --leader \
 
 ## Cleanup Portworx framework
 
+{{<info>}}
+This section presents the **DC/OS** way of cleaning up the Portworx framework. Please refer to the [Uninstall on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/operate-and-maintain/framework-cleanup.md]:
```suggestion
This section presents the **DC/OS** method of cleaning up the Portworx framework. Please refer to the [Uninstall on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/operate-and-maintain/going-production-with-dcos.md [hunk 12]
```diff
@@ -6,6 +6,10 @@ weight: 1
 linkTitle: Running Portworx in Production with DC/OS
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of running Portworx in production. Please refer to the [Running in Production](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/running-in-production/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/operate-and-maintain/going-production-with-dcos.md]:
```suggestion
This document presents the **DC/OS** method of running Portworx in production. Please refer to the [Running in Production](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/running-in-production/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/operate-and-maintain/upgrade-oci.md [hunk 13]
```diff
@@ -6,6 +6,10 @@ weight: 2
 linkTitle: Upgrade
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of upgrading Portworx. Please refer to the [Upgrade Portworx on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/operate-and-maintain/upgrade-oci.md]:
```suggestion
This document presents the **DC/OS** method of upgrading Portworx. Please refer to the [Upgrade Portworx on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/dcos/operate-and-maintain/upgrade.md [hunk 14]
```diff
@@ -5,6 +5,10 @@ description: Updating Portworx version on your DCOS cluster is simple. Follow th
 hidden: true
 ---
 
+{{<info>}}
+This document presents the **DC/OS** way of upgrading Portworx. Please refer to the [Upgrade Portworx on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/dcos/operate-and-maintain/upgrade.md]:
```suggestion
This document presents the **DC/OS** method of upgrading Portworx. Please refer to the [Upgrade Portworx on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/docker/_index.md [hunk 15]
```diff
@@ -7,9 +7,11 @@ series: px-other
 hidesections: true
 ---
 
-This section covers information on installing Portworx on a cluster that uses Docker Swarm or just vanilla Docker (no orchestrator).
+{{<info>}}
+This document presents the **Docker** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/docker/_index.md]:
```suggestion
This document presents the **Docker** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/docker/_index.md [hunk 16]
```diff
@@ -7,9 +7,11 @@ series: px-other
...
+{{</info>}}
 
-{{<info>}}If you have a cluster that uses an orchestrator like Kubernetes, DCOS etc, please choose the [appropriate page](/).{{</info>}}
+This section covers information on installing Portworx on a cluster that uses Docker Swarm or just vanilla Docker (no orchestrator).
```
Comment (nathan-px) [inline content/install-with-other/docker/_index.md]: Provides information or instructions? 

```suggestion
This section provides information on installing Portworx on a cluster that uses Docker Swarm or just vanilla Docker (no orchestrator).
```

File: content/install-with-other/docker/how-to/use-px-with-swarm.md [hunk 17]
```diff
@@ -6,6 +6,10 @@ weight: 1
 linkTitle: Use PX with Swarm
 ---
 
+{{<info>}}
+This document presents the **Docker** way of deploying stateful applications using Docker Swarm. Please refer to the [Stateful applications on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/docker/how-to/use-px-with-swarm.md]:
```suggestion
This document presents the **Docker** method of deploying stateful applications using Docker Swarm. Please refer to the [Stateful applications on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/docker/standalone/_index.md [hunk 18]
```diff
@@ -7,6 +7,10 @@ weight: 1
 series: px-docker-install
 ---
 
+{{<info>}}
+This document presents the **Docker** way of installing a Portworx cluster using `runC` containers. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]:
```suggestion
This document presents the **Docker** method of installing a Portworx cluster using `runC` containers. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/docker/stateful-applications/_index.md [hunk 19]
```diff
@@ -6,4 +6,8 @@ weight: 4
 series: px-docker-post-install
 ---
 
+{{<info>}}
+This document presents the **Docker** way of deploying stateful applications. Please refer to the [Stateful applications on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/docker/stateful-applications/_index.md]:
```suggestion
This document presents the **Docker** method of deploying stateful applications. Please refer to the [Stateful applications on Kubernetes](/portworx-install-with-kubernetes/application-install-with-kubernetes/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/docker/swarm/_index.md [hunk 20]
```diff
@@ -7,6 +7,10 @@ noicon: true
 series: px-docker-install
 ---
 
+{{<info>}}
+This document presents the **Docker** way of installing a Portworx cluster using Docker in swarm mode. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/docker/swarm/_index.md]:
```suggestion
This document presents the **Docker** method of installing a Portworx cluster using Docker in swarm mode. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/ecs/_index.md [hunk 21]
```diff
@@ -8,6 +8,10 @@ noicon: true
 series: px-other
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of installing Portworx on Amazon ECS. Please refer to the [AWS](/portworx-install-with-kubernetes/cloud/aws/) page, for details on installing Portworx with Kubernetes on AWS.
```
Comment (nathan-px) [inline content/install-with-other/ecs/_index.md]:
```suggestion
This document presents the **non-Kubernetes** method of installing Portworx on Amazon ECS. Please refer to the [AWS](/portworx-install-with-kubernetes/cloud/aws/) page for details on installing Portworx with Kubernetes on AWS.
```

File: content/install-with-other/nomad/_index.md [hunk 22]
```diff
@@ -8,6 +8,10 @@ series: px-other
 noicon: true
 ---
 
+{{<info>}}
+This document presents the **Nomad** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/_index.md]:
```suggestion
This document presents the **Nomad** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/nomad/install-as-a-nomad-job/_index.md [hunk 23]
```diff
@@ -9,6 +9,10 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents the **Nomad** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/install-as-a-nomad-job/_index.md]:
```suggestion
This document presents the **Nomad** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/nomad/install-hashi-porx/_index.md [hunk 24]
```diff
@@ -9,6 +9,10 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents a **non-Kubernetes** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/install-hashi-porx/_index.md]:
```suggestion
This document presents a **non-Kubernetes** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/nomad/install-with-ansible/_index.md [hunk 25]
```diff
@@ -9,6 +9,11 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents a **non-Kubernetes** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/install-with-ansible/_index.md]:
```suggestion
This document presents a **non-Kubernetes** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/nomad/install-with-terraform/_index.md [hunk 26]
```diff
@@ -9,6 +9,10 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents a **non-Kubernetes** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/install-with-terraform/_index.md]:
```suggestion
This document presents a **non-Kubernetes** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/nomad/post-install-uninstall-portworx/_index.md [hunk 27]
```diff
@@ -10,6 +10,11 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents the **Nomad** way of uninstalling a Portworx cluster. Please refer to the [Uninstall on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/post-install-uninstall-portworx/_index.md]:
```suggestion
This document presents the **Nomad** method of uninstalling a Portworx cluster. Please refer to the [Uninstall on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/nomad/post-install-upgrade/_index.md [hunk 28]
```diff
@@ -10,6 +10,10 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents the **Nomad** way of upgrading a Portworx cluster. Please refer to the [Upgrade Portworx on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/) page, if you want to upgrade Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/post-install-upgrade/_index.md]:
```suggestion
This document presents the **Nomad** method of upgrading a Portworx cluster. Please refer to the [Upgrade Portworx on Kubernetes](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/) page if you want to upgrade Portworx on Kubernetes.
```

File: content/install-with-other/nomad/useful-information-storage-on-demand/_index.md [hunk 29]
```diff
@@ -9,6 +9,10 @@ noicon: true
 hidden: true
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of provisioning volumes on-demand. Please refer to the [Dynamic Provisioning of PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/dynamic-provisioning/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/nomad/useful-information-storage-on-demand/_index.md]:
```suggestion
This document presents the **non-Kubernetes** method of provisioning volumes on-demand. Please refer to the [Dynamic Provisioning of PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/dynamic-provisioning/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/_index.md [hunk 30]
```diff
@@ -5,3 +5,7 @@ description: Instructions on how to operate and maintain installations with Port
 weight: 9
 series: px-other
 ---
+
+{{<info>}}
+This document presents the **non-Kubernetes** way of operating and maintaining your Portworx cluster. Please refer to the [Operate and Maintain](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/_index.md]:
```suggestion
This document presents the **non-Kubernetes** method of operating and maintaining your Portworx cluster. Please refer to the [Operate and Maintain](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/monitoring/_index.md [hunk 31]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Monitoring your cluster"
 keywords: portworx, node_exporter, cadvisor prometheus, grafana, alertmanager, cluster, storage
-description: How to monitor various custers on Portworx.
+description: How to monitor your Portworx cluster.
 weight: 2
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of monitoring a Portworx cluster. Please refer to the [Monitoring](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/_index.md]:
```suggestion
This document presents the **non-Kubernetes** method of monitoring a Portworx cluster. Please refer to the [Monitoring](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/monitoring/alerting.md [hunk 32]
```diff
@@ -4,6 +4,10 @@ keywords: portworx, prometheus, grafana, alertmanager, cluster, storage
 description: Learn how to configure Prometheus to enable you to visualize your Portworx cluster status within Grafana.
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of monitoring your Portworx cluster with Prometheus and Grafana. Please refer to the [Prometheus and Grafana](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/alerting.md]:
```suggestion
This document presents the **non-Kubernetes** method of monitoring your Portworx cluster with Prometheus and Grafana. Please refer to the [Prometheus and Grafana](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/monitoring/grafana/index.md [hunk 33]
```diff
@@ -4,6 +4,11 @@ keywords: portworx, prometheus, grafana, alertmanager, cluster, storage
 meta-description: Find templates for displaying Portworx cluster information within Grafana.
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of monitoring your Portworx cluster with Grafana. Please refer to the [Prometheus and Grafana](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/grafana/index.md]:
```suggestion
This document presents the **non-Kubernetes** method of monitoring your Portworx cluster with Grafana. Please refer to the [Prometheus and Grafana](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/monitoring/prometheus/index.md [hunk 34]
```diff
@@ -3,6 +3,10 @@ title: Portworx integration with prometheus
 keywords: prometheus, graph, stats
 meta-description: Looking to integrate Portworx with Prometheus? Learn to integrate PX storage with Prometheus for monitoring today!
 ---
+{{<info>}}
+This document presents the **non-Kubernetes** way of monitoring your Portworx cluster with Prometheus. Please refer to the [Prometheus and Grafana](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/prometheus/index.md]:
```suggestion
This document presents the **non-Kubernetes** method of monitoring your Portworx cluster with Prometheus. Please refer to the [Prometheus and Grafana](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/scaling/_index.md [hunk 35]
```diff
@@ -5,3 +5,8 @@ keywords: portworx, px-enterprise, storage, volume, scale, scaling, scaleup,scal
 description: Articles on understanding how to scale up and down a Portworx clusters
 weight: 3
 ---
+
+{{<info>}}
+This document presents the **non-Kubernetes** way of scaling your Portworx cluster. Please refer to the [Scale or Restrict](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/scale-or-restrict/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/scaling/_index.md]:
```suggestion
This document presents the **non-Kubernetes** method of scaling your Portworx cluster. Please refer to the [Scale or Restrict](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/scale-or-restrict/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/scaling/scale-down.md [hunk 36]
```diff
@@ -4,6 +4,12 @@ keywords: scale-down
 description: Find out how to remove an offline node from a cluster with this document from Portworx. Discover how to scale-down nodes today!
 ---
 
+
+{{<info>}}
+This document presents the **non-Kubernetes** way of removing a node. Please refer to the [Decommission a Node ](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/decommission-a-node/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/scaling/scale-down.md]:
```suggestion
This document presents the **non-Kubernetes** method of removing a node. Please refer to the [Decommission a Node ](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/decommission-a-node/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/scaling/scale-out.md [hunk 37]
```diff
@@ -4,6 +4,11 @@ keywords: scale-up, storage, add new nodes
 description: This document illustrates how to add a new node with attached storage to a PX cluster.  Scale out your cluster today!
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of scaling your Portworx cluster. Please refer to the [Scale or Restrict](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/scale-or-restrict/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/scaling/scale-out.md]:
```suggestion
This document presents the **non-Kubernetes** method of scaling your Portworx cluster. Please refer to the [Scale or Restrict](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/scale-or-restrict/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/operate-and-maintain/scaling/scale-up.md [hunk 38]
```diff
@@ -4,6 +4,10 @@ keywords: scale-up
 description: Discover how to add a new node to a PX cluster and how to add additional storage to the PX Cluster once a new node is added.  Try it for yourself today.
 ---
 
+{{<info>}}
+This document presents the **non-Kubernetes** way of scaling your Portworx cluster. Please refer to the [Scale or Restrict](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/scale-or-restrict/) page, if you are running Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/scaling/scale-up.md]:
```suggestion
This document presents the **non-Kubernetes** method of scaling your Portworx cluster. Please refer to the [Scale or Restrict](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/scale-or-restrict/) page if you are running Portworx on Kubernetes.
```

File: content/install-with-other/other-clouds/digitial-ocean/_index.md [hunk 39]
```diff
@@ -7,6 +7,10 @@ linkTitle: DigitalOcean
 noicon: true
 ---
 
+{{<info>}}
+This document presents a **non-Kubernetes** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/other-clouds/digitial-ocean/_index.md]:
```suggestion
This document presents a **non-Kubernetes** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/other-clouds/packet/_index.md [hunk 40]
```diff
@@ -7,6 +7,10 @@ linkTitle: Packet.net
 noicon: true
 ---
 
+{{<info>}}
+This document presents a **non-Kubernetes** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/other-clouds/packet/_index.md]:
```suggestion
This document presents a **non-Kubernetes** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

File: content/install-with-other/other-clouds/rackspace/_index.md [hunk 41]
```diff
@@ -7,6 +7,10 @@ linkTitle: Rackspace
 noicon: true
 ---
 
+{{<info>}}
+This document presents a **non-Kubernetes** way of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page, if you want to install Portworx on Kubernetes.
```
Comment (nathan-px) [inline content/install-with-other/other-clouds/rackspace/_index.md]:
```suggestion
This document presents a **non-Kubernetes** method of installing a Portworx cluster. Please refer to the [Portworx on Kubernetes](/portworx-install-with-kubernetes/) page if you want to install Portworx on Kubernetes.
```

**Comment by andreipope** [comment]
> Approved, but please accept grammar suggestions to remove the unnecessary comma and more precise word choice.

Thanks for taking the time to review it. I accepted all your suggestions.

**Pull request merged**